src/components/inputField/InputField.stories.tsx

```
@@ -16,14 +16,20 @@ type Story = StoryObj<typeof InputField>;
 export const Default: Story = {
 	args: {
 		label: "유효기간",
-		inputs: [<Input placeholder="MM" value="" onChange={() => {}} />, <Input placeholder="YY" value="" onChange={() => {}} />],
+	},
+	render: (args) => {
+		const inputs = [<Input placeholder="MM" value="" onChange={() => {}} />, <Input placeholder="YY" value="" onChange={() => {}} />];
+		return <InputField label={args.label}>{inputs}</InputField>;
 	},
 };
 
 export const Error: Story = {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)